rrfs-test/testinput/rrfs_fv3jedi_bumploc_2022052619.yaml

```diff
@@ -0,0 +1,86 @@
+geometry:
+ fms initialization:
+   namelist filename: DataFix/fmsmpp.nml
+   field table filename: DataFix/field_table
+ namelist filename: DataFix/input_lam_C775.nml
+ akbk: DataFix/fix/akbk61.nc
+ field metadata override: Data/fieldmetadata/tlei-gfs-restart.yaml
+ # input.nml
+ npz: 60
+ layout: [8,10]
+ io_layout: [1,1]
+ ntiles: 1 
+background:
+  datetime: 2022-05-26T19:00:00Z
+  filetype: fms restart
+  datapath: Data/bkg
+  filename_core: fv3_dynvars.nc
+  filename_trcr: fv3_tracer.nc
+  filename_sfcd: fv3_sfcdata.nc
+  filename_cplr: coupler.res
+  state variables: [air_temperature,surface_pressure] 
+background error:
+  covariance model: SABER
+  saber central block:
+    saber block name: BUMP_NICAS
+    calibration:
+      general:
+        universe length-scale: 250.0e3
+      io:
+        files prefix: Data/bump_new/fv3jedi_bumpparameters_nicas_lam_atm
+        alias:
+        - in code: air_temperature
+          in file: fixed_250km_0.3
+        - in code: surface_pressure
+          in file: fixed_250km
+
+      drivers:
+        multivariate strategy: univariate 
+        compute nicas: true
+        write local nicas: true
+      nicas:
+        resolution: 6
+        explicit length-scales: true
+        horizontal length-scale:
+        - groups:
+          - air_temperature
+          value: 250000.0
+        - groups:
+          - surface_pressure
+          value: 250000.0
+
+        vertical length-scale:
+        - groups:
+          - air_temperature
+          value: 0.3
+        - groups:
+          - surface_pressure
+          value: 0.0
```

Comment:
@TingLei-NOAA For line 58, does it mean no any vertical correlation for surface_pressure?

Reply:
@guoqing-noaa   Good question.  I think it means no vertical correlation for surface pressure --- the 2D control variable/analysis variables (since it is a 2D variable).   So for all 3d analysis variables, if not specified specifically like air_temperature in this case, it would use the length scales for the air_temperature (3D ) group.  All other 2d control variables like sst would use ones in the surface_pressure group.

Reply:
@TingLei-NOAA Thanks for the information. 
(1) if we don't specify "air_temperature", but use humidity, will those not-specified 3D variables use the length scale for humidity? 
(2) For some 2D variables, I understand that we only want to do 2DVAR. But for surface pressure, we may want to spread vertically to some extent. When we change settings for this, will all other 2D variables change accordingly?  Thanks!

Reply:
Guoqing,  1) Yes.   2)  here, the surface_pressure is the analysis variable , I am wondering How you would want to "spread vertically "?   For analysis using surface pressure observation,  the surface observation would spread its impact vertically to those 3d anallysis variables (since non-zero vertical localization is set up).   But, specific to your question, if you change this set up for surface pressure (analysis variable) , yes, all other 2d variable will use the changed setup ( if only one 2d group exist).  For multiple 2d group situation, we need to dig more to see what the behavior would be).

Reply:
Thanks, Ting! I will need to learn more about BUMP and NICAS. So the vertical distribution of obs innovation is through ensemble BEC, right?

If we don't have the surface_pressure group, will the 2D variables also use a default 0.0 vertical length scale?

Reply:
> Thanks, Ting! I will need to learn more about BUMP and NICAS.
Same here!
> So the vertical distribution of obs innovation is through ensemble BEC, right?
Yes!
> 
> If we don't have the surface_pressure group, will the 2D variables also use a default 0.0 vertical length scale?
I am not sure. Will the jedi Var app abort complaining about missing group?   I need to learn about them:).